identity-account/README.md

```diff
@@ -0,0 +1,70 @@
+# Iota Identity Account
+
+The Account is an interface for creating and managing identities on the Iota Tangle. It includes easy-to-use functions that allow many operations including:
+
+- Creating an identity or resolving an existing DID Document.
+- Updating a DID Document by adding or removing:
+  - Services.
+  - Verification Methods.
+  - Verification Relationships.
+  - Custom properties.
+- Managing and storing the state of DID Documents.
+- Managing and storing private encryption keys used as verification methods.
+
+## Account Creation
+
+Creating an account is done through the `AccountBuilder`.
+
+```rust
+  let account: Account = Account::builder()
+    .create_identity(IdentitySetup::default())
+    .await?;
+```
+
+Various options can be set for the `AccountBuilder` including:
+
+- `autosave`: sets the account auto-save behaviour.
+  - `AutoSave::Every` => save to storage on every update.
```

Comment:
Would be great if you provided links to items i.e. [`AutoSave`](crate::account::AutoSave)